mkosi.default.d/debian/10-mkosi.debian

```diff
@@ -5,7 +5,7 @@
 
 [Distribution]
 Distribution=debian
-Release=unstable
+Release=bullseye
```

Comment:
make this `testing` - bullseye is the stable release, but for CIs we want something newer, testing is more or less equivalent to fedora rawhide or archlinux or suse tumbleweed

Reply:
Last time I checked the "testing" tag wasn't supported (I'm not sure whether it was mkosi though). I'll change it shortly

Reply:
FWIW opensuse is flaky as well (partly because of the way the mkosi action works as far as I understand). It was briefly discussed in https://github.com/systemd/systemd/pull/21978#issuecomment-1004084735

Reply:
Anyway, judging by
```
              Distribution: debian
                   Release: testing
                    Mirror: http://deb.debian.org/debian
     Use Host Repositories: no
```
it seems to be working